graphs/eacs/src/test/kotlin/com/anaplan/engineering/azuki/graphs/eacs/BEH7.kt

```diff
@@ -146,29 +88,7 @@ class BEH7 : GraphScenario() {
         }
     }
 
-    @Eac("When a directed graph has all edges declared twice, return 1")
-    fun hasDoubleDeclaredEdgesDirected() {
-        given {
-            thereIsADirectedGraph(graphA) {
-                edge("a", "b")
-                edge("b", "c")
-                edge("d", "a")
-                edge("e", "d")
-                edge("c", "e")
-                edge("a", "b")
-                edge("b", "c")
-                edge("d", "a")
-                edge("e", "d")
-                edge("c", "e")
-            }
-        }
-        then {
-            hasCycleCount(graphA, 1)
-        }
-    }
-
-    //TODO() figure out if this is operating correctly cause this looks strange to me.
-    @Eac("When a directed graph has two vertices which have a edge in both directions, return 1")
+    @Eac("When a given graph has two vertices which have a edge in both directions, return 1")
```

Comment:
🚫 Similar comments apply here. I would phrase it like

> A cycle can constitute of two vertices and two edges between them, in opposite directions

This is borderline an AC, but it might be worth keeping. If you do choose to keep it, BEH6 should have the same AC.